src/res_core.ml

```diff
@@ -2946,7 +2951,7 @@ and parseRecordRow p =
       let fieldExpr = parseExpr p in
       Some (field, fieldExpr)
     | _ ->
-      Some (field, Ast_helper.Exp.ident ~loc:field.loc  field)
+      Some (field, removeModuleNameFromPunnedFieldValue (Ast_helper.Exp.ident ~loc:field.loc field))
```

Comment:
I'm wondering if it would make sense to only run the `removeModuleNameFromPunnedFieldValue` name if we started from the `Uident` branch. What do you think?

Reply:
Done

Reply:
Oops, I see there is still something wrong here.

Reply:
Now it looks good.